### PR TITLE
feat: run at specific days and times

### PR DESCRIPTION
**Pull request by skye2k2**
Add the ability to run on specific days and times, as opposed to on a timeout since the server started (which could get weird after midnight Heroku restarts or deploys), and some teams don't work weekends. Also add labels to explicitly exclude results.

Add Moment.js dependency.
Add DAYS_TO_RUN, TIMES_TO_RUN, TZ, GH_EXCLUDE_LABELS, and DEBUG environment variables, with default configurations. Remove CHECK_INTERVAL.
Utilize Moment.js to get current date and automatically determine what locale DAYS_TO_RUN should be checked against.
Utilize Moment.js to check current time against TIMES_TO_RUN.
Add logic to exclude pull-requests which have specified labels.
Gate additional logging by DEBUG flag.
Add new properties to README.md and app.json (alphabetize properties).
Update .gitignore to exclude .env file.

### CODE REVIEW

**Comment by skye2k2** [comment]
@rogeriopvl: I think this _should_ work without having to set a LOCALE environment variable or parameter for Moment.js. Could someone from your side check?

**Comment by rogeriopvl** [comment]
Hey @skye2k2, thanks for the pull request. I'll have a look at it asap 👍

**Comment by skye2k2** [comment]
So now the server is now restart-proof, but I have broken the unit test `calls pullhub on start`, since we would now check for PR's on a much more restricted basis, not just a timeout. Disabled, for now.

**Comment by skye2k2** [comment]
`sudo add tag: "needs review"`

**Comment by skye2k2** [comment]
@rogeriopvl: Any chance this will work for you? Our team has been using this version for the past month to great effect.

**Comment by FalconerTC** [comment]
What's the status of this? Looks like a nifty feature

**Comment by skye2k2** [comment]
[This](https://github.com/fs-webdev/pr-police) incorporates the latest and greatest of our organization's customizations on top of @rogeriopvl 's work (most notably holiday exclusion, and includes the modifications in this PR), and has worked extremely well over the past 9 months.

I find it highly ironic that _this_ PR has languished for so long...

**Comment by drwpow** [comment]
@skye2k2 just deployed your branch to our org and it works like a charm! Definitely hoping this gets merged. It’s been nothing but helpful, especially getting to set it to chime in at specific times and weekdays only.

**Comment by diogoosorio** [comment]
@sryabkov although we're running this code internally at Talkdesk (multiple instances of it in a matter of fact), the project was internally pretty much abandoned / neglected since @rogeriopvl left the company.

~My team was allowed to take over the project - I'm going to merge this PR for you, I hope you don't mind me doing so. 🙏~

Thanks for the contribution and sorry for the very (very) late feedback. 😅

**Comment by diogoosorio** [comment]
Didn't see the conflict on the README file. Can you please rebase with master so that we can move this PR along? 🙏

**Comment by skye2k2** [comment]
@diogoosorio: The remaining issue is that momentjs is not part of the `package.lock` that was added after this PR was created. As you are already working on updating dependencies (https://github.com/Talkdesk/pr-police/pull/15), I have given maintainers write access to this branch.

**Comment by rogeriopvl** [comment]
@skye2k2 yeah, sorry for not replying sooner. I no longer have permissions to merge PRs on this repo. But it's great to see that @diogoosorio picked this up 🎉

**Comment by sryabkov** [comment]
@diogoosorio My change was marking which parameters are required and which ones are optional. I am reluctant to go messing with README as other changes have been made, and new parameters seem to have been added. I would not know what to put in for them. If you simply want to remove my changes, that's fine too.

**Comment by diogoosorio** [comment]
I've tweaked the implementation a bit (but the contract/configuration you've introduced still stands). @dcbarbosa / @jrebocho can one of you guys give the changes a quick look so that we can merge this?

File: README.md [hunk 1]
```diff
@@ -53,8 +53,8 @@ The github account token to access the repos. Required.
 ##### `SLACK_TOKEN`
 The slack token for the bot to access your slack team. Required.
 
-#### `GH_REPOS`
-The list of repositories to watch. The format is `user/repo` and comma separated. Required.
+##### `GH_REPOS`
+The list of repositories to watch. The format is `user/repo` and comma separated.
```
Comment (dcbarbosa) [inline README.md]: Isn't this field required?

**Review by dcbarbosa — APPROVED** [review]
LGTM 👍 I only left a small comment.

**Pull request merged**